drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -387,12 +387,7 @@ function DragAndDropTemplates(configuration) {
             popupSelector,
             {
                 style: {display: have_messages ? 'block' : 'none'},
-                attributes: {
-                    "tabindex": "-1",
-                    'aria-live': 'polite',
-                    'aria-atomic': 'true',
-                    'aria-relevant': 'all',
-                }
+                attributes: {"tabindex": "-1"}
```

Comment:
I don't understand either. It looks like it was added as part of a larger patch and might not have been intentional. I removed it and cannot see any issues.